awx/ui_next/src/screens/User/shared/UserForm.test.jsx

```diff
@@ -0,0 +1,157 @@
+      wrapper = mountWithContexts(
+        <UserForm handleSubmit={jest.fn()} handleCancel={jest.fn()} />
+      );
+    });
+    await waitForElement(wrapper, 'ContentLoading', el => el.length === 0);
+    expect(wrapper.find('FormGroup[label="Username"]').length).toBe(1);
+    expect(wrapper.find('FormGroup[label="Email"]').length).toBe(1);
+    expect(wrapper.find('FormGroup[label="First Name"]').length).toBe(1);
+    expect(wrapper.find('FormGroup[label="Last Name"]').length).toBe(1);
+    expect(wrapper.find('FormGroup[label="Password"]').length).toBe(1);
+    expect(wrapper.find('FormGroup[label="Confirm Password"]').length).toBe(1);
+    expect(wrapper.find('FormGroup[label="Organization"]').length).toBe(1);
+    expect(wrapper.find('FormGroup[label="Job Type"]').length).toBe(1);
+  });
+
+  test('edit form hides org field', async () => {
+    await act(async () => {
+      wrapper = mountWithContexts(
+        <UserForm
+          handleSubmit={jest.fn()}
+          handleCancel={jest.fn()}
+          user={mockData}
+        />
+      );
+    });
+    await waitForElement(wrapper, 'ContentLoading', el => el.length === 0);
+    expect(wrapper.find('FormGroup[label="Organization"]').length).toBe(0);
+  });
+
+  test('inputs should update form value on change', async () => {
+    await act(async () => {
+      wrapper = mountWithContexts(
+        <UserForm handleSubmit={jest.fn()} handleCancel={jest.fn()} />
+      );
+    });
+    await waitForElement(wrapper, 'ContentLoading', el => el.length === 0);
+    const form = wrapper.find('Formik');
+    act(() => {
+      wrapper.find('OrganizationLookup').invoke('onBlur')();
+      wrapper.find('OrganizationLookup').invoke('onChange')({
+        id: 1,
+        name: 'organization',
+      });
+    });
+    expect(form.state('values').organization).toEqual(1);
+  });
+
+  test('password fields are required on add', async () => {
+    await act(async () => {
+      wrapper = mountWithContexts(
+        <UserForm handleSubmit={jest.fn()} handleCancel={jest.fn()} />
+      );
+    });
+
+    const passwordFields = wrapper.find('PasswordField');
+
+    expect(passwordFields.length).toBe(2);
+    expect(passwordFields.at(0).prop('isRequired')).toBe(true);
+    expect(passwordFields.at(1).prop('isRequired')).toBe(true);
+  });
+
+  test('password fields are not required on edit', async () => {
+    await act(async () => {
+      wrapper = mountWithContexts(
+        <UserForm
+          user={mockData}
+          handleSubmit={jest.fn()}
+          handleCancel={jest.fn()}
+        />
+      );
+    });
+
+    const passwordFields = wrapper.find('PasswordField');
+
+    expect(passwordFields.length).toBe(2);
+    expect(passwordFields.at(0).prop('isRequired')).toBe(false);
+    expect(passwordFields.at(1).prop('isRequired')).toBe(false);
+  });
+
+  test('should call handleSubmit when Submit button is clicked', async () => {
+    const handleSubmit = jest.fn();
+    await act(async () => {
```

Comment:
In the interest of consistency can we put `handleSubmit= jest.fn()` back on line 129?